datafusion/functions/Cargo.toml

```diff
@@ -75,6 +75,8 @@ log = { workspace = true }
 md-5 = { version = "^0.10.0", optional = true }
 regex = { version = "1.8", optional = true }
 sha2 = { version = "^0.10.1", optional = true }
+uuid = { version = "1.7", features = ["v4"] }
```

Comment:
I think technically this could be an optional depdendece (on string expressions) but since datafusion-core relies on uuid too. I'll try and clean it up as a follow on

Reply:
https://github.com/apache/arrow-datafusion/pull/9771